src/blog/pointer-compression.md

```diff
@@ -206,24 +207,40 @@ Let’s take a look at the x64 assembly.
 |               | andl r10,0x1            | jz done                      \
 |               | negq r10                | addq r11,r13                 \
 |               | andq r10,r13            | done:                        \
-|               | addq r11,r10            | ```                          \
-|               | ```                     |                              |
+|               | addq r11,r10            |  ```                         \
+|               | ```                     |  <br>                        |
```

Comment:
Hmm what happened here / why does this need `<br>` again?

Reply:
(same for other tables - we shouldn't need it anymore)

Reply:
br is there so that both code blocks start at the top

Reply:
Hmm I'm conflicted about it, but I guess fine. Just as another option, would leaving a newline inside the code block help? (then both code blocks would have the same size, which might be even better visually for showing the difference)

Reply:
That's what I tried first, but it does not work (the new line is removed)

Reply:
Hmm it works on my side. To be clear - I'm talking about an extra newline before the closing ```` ``` ````.

Reply:
@santiaboy Ping - what do you think?

Reply:
Nvm, this one doesn't work.

Reply:
Found how this one worked - you just need to use `| \` in the end of the line with spaces instead of `\`.

@santiaboy can you please try again?